CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## 🎉 Features 🎉
+- Added ability for a user to allow records having nulls in url fields to be included in the `ad_reporting__url_report` and/or the underlying `...url_report` models. This is done by setting the below variable to `False` in your `dbt_project.yml` file. 
+```yml
+vars:
+  ad_reporting__url_report__using_null_filter: False # Use this variable to allow null urls for ALL Fivetran ad packages included in your project. Default is True. 
+```
+- Updated README with this information. 
+
+## 🚘 Under the Hood 🚘
+- Disabled the `not_null` test for `ad_reporting__url_report` when null urls are allowed. [#72](https://github.com/fivetran/dbt_ad_reporting/pull/72)
+
+
```

Comment:
remember to move the above section into the migrations changelog entry!

Reply:
done